### ISSUE TITLE
Right-click menu on robots tree-view 

### ISSUE THREAD

**Issue by czalidis** [stdr-simulator-ros-pkg/stdr_simulator#151]
You should be able to access the same menu either by right-clicking directly on a robot, or by right-clicking on its corresponding name appearing on the left-handed tree-view. This should be fairly easy to implement.

**Comment by Scalpel78** [comment]
Agreed. I'm just starting out with STDR, and find the GUI really confusing.
